### PR TITLE
BUG 1853785: Retry for VF link to become ready

### PR DESCRIPTION
**Pull request by zshi-redhat**
VF link may not be immediately ready after VF creation.

### CODE REVIEW

File: pkg/utils/utils.go [hunk 1]
```diff
@@ -460,10 +474,15 @@ func setVfsAdminMac(iface *sriovnetworkv1.InterfaceExt) error {
 			glog.Errorf("setVfsAdminMac(): unable to get VF id %+v %q", iface.PciAddress, err)
 			return err
 		}
+		_, err = vfIsReady(addr)
+		if err != nil {
+			glog.Errorf("setVfsAdminMac(): VF link is not ready for device %+v %q", addr, err)
+			return err
+		}
 		vfName := tryGetInterfaceName(addr)
```
Comment (pliurh) [inline pkg/utils/utils.go:482]: As we've got the vf name in `vfIsReady`, can we remove this line?  And also for the `vfLink` line bellow.

Comment (zshi-redhat) [reply pkg/utils/utils.go]: fixed

**Pull request merged**